addons/event_booth/models/event_event.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo import Command
+
+
+class Event(models.Model):
+    _inherit = 'event.event'
+
+    event_booth_category_ids = fields.One2many(
+        'event.booth.category', 'event_id', string='Booth Category',
+        compute='_compute_event_booth_category_ids', readonly=False, store=True)
+    event_booth_ids = fields.One2many(
+        'event.booth', 'event_id', string='Booths')
+    event_booth_count = fields.Integer(
+        string='Total Booths',
+        compute='_compute_event_booth_count', store=True)
+    event_booth_count_available = fields.Integer(
+        string='Available Booths',
+        compute='_compute_event_booth_count', store=True)
+    event_booth_slot_ids = fields.One2many(
+        'event.booth.slot', 'event_id', string='Slots')
+    event_booth_slot_count = fields.Integer(
+        string='Total Slots',
+        compute='_compute_event_booth_slots_count', store=True)
+    event_booth_slot_count_available = fields.Integer(
+        string='Available Slots',
+        compute='_compute_event_booth_slots_count', store=True)
+
+    @api.depends('event_type_id')
+    def _compute_event_booth_category_ids(self):
```

Comment:
a small docstring can be helpful for my poor brain :smile: 
why is it needed to be readonly=False, store=True ?

Reply:
I put it readonly=False so the event manager can add another specific event.booth.category to the event.